_includes/ee-linux-install-reuse.md

```diff
@@ -144,11 +144,8 @@ You can install Docker EE in different ways, depending on your needs:
     This example uses the `sort -r` command to sort the results by version
     number, highest to lowest, and is truncated.
 
-    > **Note**: This `yum list` command only shows binary packages. To show
-    > source packages as well, omit the `.x86_64` from the package name.
-
     ```bash
-    $ sudo yum list docker-ee.x86_64  --showduplicates | sort -r
+    $ sudo yum list docker-ee  --showduplicates | sort -r
```

Comment:
Is this just in case you actually are running s390x?

Reply:
That, too (or any other non-x86_64 architecture supported, like arm, aarch64, ppc64 etc.)

Also, as I explain in commit message, since source repos are disabled (by default), the only packages one will see will be docker-ee.x86_64. Therefore, specifying architecture is redundant, there is no need to.